app/assets/stylesheets/application.css

```diff
@@ -10,7 +10,9 @@
  * files in this directory. Styles in this file should be added after the last require_* statement.
  * It is generally better to create a new file per style scope.
  *
- *= require_tree .
```

Comment:
Can you comment on why we want to remove the `require_tree` directive? Will there be assets in the path that we don't want to require?

http://guides.rubyonrails.org/asset_pipeline.html#manifest-files-and-directives

Reply:
I think it's good to be explicit about what exactly is being loaded. Doing that loads them in an order that we may not want, and if we need to reorder them later it will be easier if we've been doing it from the beginning. I'm planning on doing all the customizations through `mahonia.scss`. So if we wanted to go back to the default we could just remove that line. 

Also, the Hyrax/Hyku folks do it this way: 
https://github.com/samvera-labs/hyku/blob/master/app/assets/stylesheets/application.css

Reply:
Great. In the future it would be handy to put information like this in commit messages so the changes can be understood with, e.g. `git blame`.